### ISSUE TITLE
`directory does not exist` warning in //oak/proto:grpc_encap_proto

### ISSUE THREAD

**Issue by tiziano88** [project-oak/oak#255]
From build logs:

```
INFO: From Generating C++ proto_library //oak/proto:grpc_encap_proto:
external/com_google_protobuf: warning: directory does not exist.
```

**Comment by tiziano88** [comment]
See https://pantheon.corp.google.com/cloud-build/builds/700d7aea-6799-4822-afa1-ec3758e88faf;step=1?project=oak-ci for actual logs.

**Comment by daviddrysdale** [comment]
There's a whole bunch of 'directory does not exist' warnings there (attached below).   The Python ones look like https://github.com/protocolbuffers/protobuf/issues/6049, and the others are somewhat similar to 
https://github.com/grpc/grpc/issues/20396.

```
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/compiler/plugin_pb2.py [for host]:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/any_pb2.py [for host]:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/timestamp_pb2.py [for host]:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/api_pb2.py [for host]:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/source_context_pb2.py [for host]:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/type_pb2.py [for host]:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/duration_pb2.py [for host]:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/empty_pb2.py [for host]:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/wrappers_pb2.py [for host]:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/field_mask_pb2.py [for host]:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/descriptor_pb2.py [for host]:
external/com_google_protobuf/python: warning: directory does not exist.
INFO: From ProtoCompile external/com_google_protobuf/python/google/protobuf/struct_pb2.py [for host]:
external/com_google_protobuf/python: warning: directory does not exist.
[1,459 / 1,747] Compiling external/com_google_asylo/asylo/identity/identity_expectation_matcher.cc; 0s remote-cache ... (8 actions, 7 running)
INFO: From Generating C++ proto_library //third_party/google/rpc:status_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating C++ proto_library //oak/proto:grpc_encap_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating C++ proto_library //oak/proto:storage_channel_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating C++ proto_library //oak/proto:storage_proto:
external/com_google_protobuf: warning: directory does not exist.
oak/proto/storage.proto:21:1: warning: Import third_party/google/rpc/status.proto but not used.
INFO: From Action oak/proto/storage.grpc.pb.h:
oak/proto/storage.proto:21:1: warning: Import third_party/google/rpc/status.proto but not used.
```

**Comment by KeithMoyer** [comment]
(for context: I wrote both of those referenced bugs)

FYI: Since you don't use py_proto_library in your code, the python-related "directory does not exist" warnings (protocolbuffers/protobuf#6049) are likely exclusively coming from Asylo. It so happens that we're almost done with a refactor which will result in our only use of py_proto_library being removed. So, those warnings will go away soon "on their own".

I did some initial analysis of the gRPC-related protobuf "directory does not exist" warnings in grpc/grpc#20396. If you're inclined to investigate these warnings further (yourself or pressing on upstream), you're probably best to start with that bug even if they're fewer in number than the python warnings.

**Comment by daviddrysdale** [comment]
@KeithMoyer many thanks for the pointers!

**Comment by daviddrysdale** [comment]
After #326 we now just have a few instances of `_google_protobuf: warning: directory does not exist.` left.

**Comment by daviddrysdale** [comment]
Seems fixed, probably since 41fb39fdc10b781be59b34e967f07724143c351d
